coral/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```

Comment:
I assume this file was created by `msw init`?

Reply:
Yes!